osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs

```diff
@@ -37,6 +37,11 @@ public class OsuDifficultyHitObject : DifficultyHitObject
         /// </summary>
         public double? Angle { get; private set; }
 
+        /// <summary>
+        /// Milliseconds elapsed since the start time of the previous <see cref="OsuDifficultyHitObject"/>, with a minimum of 50ms.
```

Comment:
Wait, isn't it
```suggestion
        /// Milliseconds elapsed since the start time of the previous <see cref="OsuDifficultyHitObject"/>, with a minimum of 25ms.
```
as per this lower down?
https://github.com/ppy/osu/blob/671fe772136671189cd404d6d9f723dc698050ad/osu.Game.Rulesets.Osu/Difficulty/Preprocessing/OsuDifficultyHitObject.cs#L51-L52